src/main/java/io/vertx/core/shareddata/LocalMap.java

```diff
@@ -30,7 +30,7 @@
  * This ensures there is no shared access to mutable state from different threads (e.g. different event loops) in the
  * Vert.x instance, and means you don't have to protect access to that state using synchronization or locks.
  * <p>
- * Your own objects can be marked as immutable by implementing the {@link Shareable} interface.
+ * Custom key and value objects should implement {@link Shareable} interface.
```

Comment:
Could you please rather rephrase all the 3 paragraphs? The 2 previous ones will be confusing after this change.